daemon/images/service.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/docker/docker/layer"
 	dockerreference "github.com/docker/docker/reference"
 	"github.com/docker/docker/registry"
+	"github.com/docker/libtrust"
```

Comment:
FWIW; had a chat with @dmcgowan, and there's a desire to get rid of this dependency; not sure if it's worth doing so as part of this PR (if we plan to remove it in the next release); his suggestion was to copy the parts we use into this repository (somewhere in `pkg` or `internal` perhaps?), so that we can remove the dependency

Reply:
@thaJeztah Sure I can do that in a followup, don't want to add more changes to this PR which is already big.